src/models/oryp11/internal-overview.md

```diff
@@ -0,0 +1,30 @@
+# Oryx Pro (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- Fan connectors are highlighted in cyan
+- Webcam/microphone connector is highlighted in yellow
+    - Webcam/microphone connector is underneath the heatsink
+- M.2 SSDs are highlighted in maroon
+    - Both slots support PCIe NVMe Gen 4
+- Wireless card is highlighted in purple
+- CMOS battery connector is highlighted in pink
+    - CMOS battery is underneath the outermost SSD
```

Comment:
"outermost" should probably be "lower" since they're vertical to each other, right?

Reply:
I was avoiding the use of upper/lower because that's dependent on which direction they have it rotated in, so it's arbitrary. I can just use the slot number and say closest to the battery if that would be better.

Reply:
I think the slot number would be better, yeah. Even from a vertical perspective, upper/lower isn't super clear on this machine since both drives are pretty close to the center.